pkg/controllermanager/controller/shoot/hibernation/reconciler.go

```diff
@@ -115,8 +115,8 @@ func (r *Reconciler) Reconcile(ctx context.Context, request reconcile.Request) (
 
 	parsedSchedules, err := parseHibernationSchedules(schedules)
 	if err != nil {
-		log.Info("Invalid hibernation schedules, stopping reconciliation")
-		return reconcile.Result{}, nil //lint:ignore nilerr We want to stop reconciliation in case of invalid hibernation schedules
+		log.Info("Invalid hibernation schedules, stopping reconciliation", "error", err)
```

Comment:
Why not `log.Error(err, "Invalid hibernation schedules, stopping reconciliation")? 😕

Reply:
Sure.